bin/create

```diff
@@ -30,17 +32,22 @@ if (args['--help'] || args._.length === 0) {
     console.log('    <template_path>: Path to a custom application template to use');
     console.log('    --shared will use the CordovaLib project directly instead of making a copy.');
     console.log('    --xwalk-shared-library will use Crosswalk shared mode to package the project that without xwalk so library.');
-    console.log('    --xwalk-apk-url configure the download URL of the Crosswalk runtime library.');
+    console.log('    --xwalk-apk-url configure the download URL of the Crosswalk runtime library, it requires --xwalk-shared-library.');
     process.exit(1);
 }
-if (args['--xwalk-shared-library'] || args['--xwalk-apk-url']) {
-    if (args['--shared']) {
-        console.log('    --shared do not support --xwalk-shared-library and --xwalk-apk-url.');
-        process.exit(1);
-    } else if (!(args['--xwalk-shared-library'] && args['--xwalk-apk-url'])) {
-        console.log('    --xwalk-shared-library and --xwalk-apk-url must be used at the same time.');
-        process.exit(1);
-    }
+
+if (typeof apk_url != 'undefined' &&
```

Comment:
Is this part of the check really needed? I thought it was covered by the other one (if `typeof apk_url` is not `string` as is the case for `undefined` the message below will be shown and the program will exit).